### ISSUE TITLE
Feature request: faster runtime

### ISSUE THREAD

**Issue by calebAtIspot** [CircleCI-Public/path-filtering-orb#106]
## Describe Request:
Would be great if path-filtering-orb ran faster. 

## Examples:
Currently set parameters takes 7 seconds. Combined with the checkout step this can go past a person's attention span ~ https://www.whitehutchinson.com/news/lenews/2017/august/article105.shtml

## Supporting Documentation Links:
maybe skip venv creation for extra speed?
Also relevant: https://github.com/CircleCI-Public/path-filtering-orb/issues/51

**Comment by calebAtIspot** [comment]
This issue is kind of vague so I guess a goal could be getting set parameters runtime to ~6 seconds.

**Comment by wyardley** [comment]
Using a smaller executor might also help? It's surprising there's no slim or alpine variant for `cimg/node`, but really, all this job probably needs is Python and git?

**Comment by marboledacci** [comment]
This should run faster now that python is not required.

**Comment by wyardley** [comment]
@marboledacci true, but this is a good reminder that #114 didn't update the executor... I opened #123 as a suggestion (could use a different tag there if preferred).

**Comment by marboledacci** [comment]
I will consider your suggestion for the next release to set something more basic, but for now is not critical as we also added a way to set any executor.
